flashcards/backend/config/urls.py

```diff
@@ -16,14 +16,15 @@
 import debug_toolbar
 
 from django.conf import settings
+from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
 
 # TODO: change admin endpoint for production and reverse proxy
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-]
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
@ToJestKrzysio we are serving staticfiles for django through nginx proxy it can also be done the same way with media files. I know you have objections doing it this way but I can't think of sth else. Any thoughts?

https://testdriven.io/blog/dockerizing-django-with-postgres-gunicorn-and-nginx/#media-files

Reply:
@bartoszbaran2 I've talked to Krzysiek yesterday and for now, we will serve media through proxy. The link above has a way to do it.

Reply:
@bartoszbaran2 I still don't see serving media throug proxy

Reply:
implemented